### PR TITLE
April VM images, update VirtualBox download

### PR DESCRIPTION
**Pull request by decklin**
If this looks good, merge and I'll upload the image.

### CODE REVIEW

**Comment by decklin** [comment]
Rebased due to CI failing because of a syntax error in master that I just fixed. Should pass now.

**Comment by gabebw** [comment]
Have we tested these out on Windows?

**Comment by eudaimonious** [comment]
Downloading on Windows 7 now. Will update you.

**Comment by eudaimonious** [comment]
This link is 404'ing: http://downloads.railsbridge.org/railsbridgevm-2015-04.box
So I'm downloading from http://downloads.railsbridge.org/railsbridgevm-2015-04-rc1.box

File: sites/en/installfest/set_up_virtual_machine.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Move into your workspace directory:
 
 Type this:
 
-    vagrant box add railsbridge201501 ./railsbridgevm-2015-01.box
+    vagrant box add railsbridge201501 ./railsbridgevm-2015-04.box
```
Comment (eudaimonious) [inline sites/en/installfest/set_up_virtual_machine.md]: you'll want to change the folder name too yea? from railsbridge201501 to railsbridge201504

File: sites/en/downloads/downloads.md [hunk 2]
```diff
@@ -30,15 +30,15 @@ Find the column for your OS, and download each file.
   <th>Linux (Ubuntu/Fedora)</th>
 </tr>
 <tr>
-  <td><a href="http://downloads.railsbridge.org/railsbridgevm-2015-01.box">January 2015 VM Image</a></td>
-  <td><a href="http://downloads.railsbridge.org/railsbridgevm-2015-01.box">January 2015 VM Image</a></td>
-  <td><a href="http://downloads.railsbridge.org/railsbridgevm-2015-01.box">January 2015 VM Image</a></td>
-  <td><a href="http://downloads.railsbridge.org/railsbridgevm-2015-01.box">January 2015 VM Image</a></td>
+  <td><a href="http://downloads.railsbridge.org/railsbridgevm-2015-04.box">April 2015 VM Image</a></td>
```
Comment (eudaimonious) [inline sites/en/downloads/downloads.md:33]: 404's

File: sites/en/installfest/set_up_virtual_machine.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Move into your workspace directory:
...
 
 Then type:
```
Comment (eudaimonious) [inline sites/en/installfest/set_up_virtual_machine.md:37]: lines 38 and 42:
vagrant init railsbridge201504

**Comment by eudaimonious** [comment]
I believe we should be instructing Windows 7 folks to use Git Bash rather than GitHub Shell. I'll make the suggested updates in another branch.

**Comment by eudaimonious** [comment]
Still unable to copy/paste into vm shell. Noticed this in the past couple workshops as well.

**Comment by gabebw** [comment]
Thank you for looking into this, Margaret.

**Comment by eudaimonious** [comment]
other than my comments above, it's ready to roll. got through the whole curriculum in windows and opened a separate PR for Git Bash use. Do we need to spend time updating the USB's for that?

**Comment by decklin** [comment]
Thanks, I will update the box names.

Sorry, should have been more clear that the files are not on the downloads site yet (so 404) but you can use the -rc1 now.

**Pull request merged**